scss/styles.scss

```diff
@@ -0,0 +1,3 @@
+@import 'vanilla-framework/scss/base';
+@include vf-b-code;
+@include vf-b-button;
```

Comment:
Can we not include the copy to clipboard icon from vanilla instead are adding it inline.